flask/flask-common/src/main/java/net/corda/flask/common/LockFile.java

```diff
@@ -24,21 +25,22 @@ private static RandomAccessFile createLockFile(Path path) throws IOException {
     @SneakyThrows
     public static LockFile acquire(Path path, boolean shared) {
         RandomAccessFile raf = createLockFile(path);
-        return new LockFile(raf.getChannel().lock(0L, Long.MAX_VALUE, shared));
+        return new LockFile(raf, raf.getChannel().lock(0L, Long.MAX_VALUE, shared));
     }
 
     @SneakyThrows
     public static LockFile tryAcquire(Path path, boolean shared) {
         RandomAccessFile raf = createLockFile(path);
         FileLock lock = raf.getChannel().tryLock(0L, Long.MAX_VALUE, shared);
         if(lock != null)
-            return new LockFile(lock);
+            return new LockFile(raf, lock);
         else return null;
```

Comment:
I'm sure this can be rewritten as:
```java
return (lock != null) ? new LockFile(lock) : null;
```
This is neater than having an `if-else` block without any `{}`.